app/models/split.rb

```diff
@@ -17,13 +17,50 @@ class Split < ActiveRecord::Base
   validate :registry_weights_must_be_integers
   validate :registry_must_have_winning_variant_if_decided
 
+  enum platform: %i(mobile desktop)
+
   before_validation :cast_registry
 
+  scope :for_presentation, ->(app_build: nil) do
```

Comment:
Where will we actually accept the `app_build` parameter?

Reply:
Probably right in the controller unless I build a decorator that takes the request bits and plumbs them in